Releasenotes.md

```diff
@@ -0,0 +1,70 @@
+- De Historie-endpoints en de bijbehorende componenten zijn uit de specificatie verwijderd en worden in een aparte API opgenomen.
+- Enkele parameternamen zijn aangepast vanwege consistentie met andere Haal-Centraal API's en/of de Design Decision om parameternamen in camelCase op te nemen
+  - verblijfplaats__gemeentevaninschrijving  --> verblijfplaats__gemeenteVanInschrijving
+  - verblijfplaats__naamopenbareruimte --> verblijfplaats__straat
+  - verblijfplaats__identificatiecodenummeraanduiding --> verblijfplaats__nummeraanduidingIdentificatie
+  - inclusiefoverledenpersonen --> inclusiefOverledenPersonen
+- Enkele namen van schema-componenten zijn aangepast vanwege consistentie met andere Haal-Centraal API's
+  - Hal-componenten die alleen _linked bevatten zijn omgenoemd van xxxHal naar xxxHalBasis
+  - Ouder_links --> OuderLinks
+  - Kind_links --> KindLinks
+  - Partner_links --> PartnerLinks
+  - AangaanHuwelijkInOnderzoek --> AangaanHuwelijkPartnerschapInOnderzoek
+  - IngeschrevenPersoon_links --> IngeschrevenPersoonLinks
+  - IngeschrevenPersoon_embedded --> IngeschrevenPersoonEmbedded
+- Schema-component Burgerservicenummer is verwijderd
+- Schema-component BinnenlandsAdres is verwijderd
+- Schema-component Verblijfbuitenlad is verwijderd. Properties zijn opgenomen in Verblijfplaats.
+- Verblijfplaats is aangepast door hergebruik van het BAG-adres
+- Inline gedefinieerde objecten zijn nu als component gedefinieerd.
+- Enkele propertynamen zijn aangepast vanwege consistentie met andere Haal-Centraal API's.
+  - IngeschrevenPersoon.nationaliteit --> IngeschrevenPersoon.nationaliteiten
+  - IngeschrevenPersoon.reisdocumenten --> IngeschrevenPersoon.reisdocumentnummers
+  - Verblijfplaats.straatnaam --> Verblijfplaats.korteNaam
+  - BinnenlandsAdres.openbareRuimteNaam --> Verblijfplaats.straat
+  - BinnenlandsAdres.woonplaatsnaam --> Verblijfplaats.woonplaats
+  - BinnenlandsAdres.identificatiecodeNummeraanduiding --> Verblijfplaats.nummeraanduidingIdentificatie  
+  - Verblijfplaats.identificatiecodeAdresseerbaarObject --> Verblijfplaats.adresseerbaarObjectIdentificatie  
+  - In de component verblijfplaatsInOnderzoek zijn de bovenstaande wijzigingen in property-namen ook doorgevoerd in de corresponderende booleans
+  - IngeschrevenPersoon_Links.verblijfplaatsNummeraanduiding --> IngeschrevenPersoonLinks.adres
+
+
+### Features:
+
+In het algemeen is de testdata verwijderd uit de feature-files. De feature files hebben als doel om de functioneel de werking te beschrijven.
+
+- adres.feature is toegevoegd
+- enum_waarden.feature is verplaatst naar [Common](https://github.com/VNG-Realisatie/Haal-Centraal-common/tree/master/features)
+- expand.feature is verplaatst naar [Common](https://github.com/VNG-Realisatie/Haal-Centraal-common/tree/master/features)
+- fields.feature is verplaatst naar [Common](https://github.com/VNG-Realisatie/Haal-Centraal-common/tree/master/features)
```

Comment:
bij fields is er een beperkte en uitgebreide variant van fields. Voor BRP wordt de uitgebreide variant gebruikt.

Reply:
Ik heb aangegeven dat field_extensie een nieuwe feature is. Moet ik dat nog verder toelichten ?

Reply:
nee, laat maar, die zag ik later pas